budget-app/src/style/theme/theme.js

```diff
@@ -6,6 +6,7 @@ const theme = {
     lightPurple: '#5142a7',
     darkPurple: '#4333a0',
     darkBlue: '#080B8D',
+    mediumRed: '#FF0000',
```

Comment:
change the obj stracture to something like this :
colors:{
purple :{
lightPurple: '#5142a7',
darkPurple: '#4333a0',
}

Reply:
better like Isaac said